src/reynir/bintokenizer.py

```diff
@@ -289,6 +289,49 @@ def all_except_suffix(s):
 # Given names that can also be family names (and thus gender- and caseless as such)
 BOTH_GIVEN_AND_FAMILY_NAMES = frozenset(("Hafstein",))
 
+# Note: these must have a meaning for this to work, so specifying them
+# as abbreviations to Main.conf is recommended
+_CORPORATION_ENDINGS = frozenset(
```

Comment:
Maybe add "hses." for [húsnæðissjálfseignastofnun](https://www.ils.is/stofnframlog/husnaedissjalfseignastofnun/).

Reply:
Done!